api/core/v1alpha1/crds/crd.go

```diff
@@ -46,6 +50,7 @@ func GetAll(k8sVersion string) (map[string]string, error) {
 	return map[string]string{
 		ResourceNameEtcd:                selectedEtcdCRD,
 		ResourceNameEtcdCopyBackupsTask: etcdCopyBackupsTaskCRD,
+		ResourceNameEtcdOpsTask:         etcdOpsTaskCRD,
```

Comment:
Since we have CEL validations defined in `EtcdOpsTask` CRD, we should maintain a non-CEL version of the CRD and return that if k8s version is below 1.29, similar to the `Etcd` CRD. Unless we enforce that druid (and EtcdOpsTask) deterministically works only for k8s v1.29 and above, which might not be a very good idea.

WDYT?